midas3.py

```diff
@@ -0,0 +1,32 @@
+from midas import Terrorist, Organization, Event, session
```

Comment:
When you're making this project into a package, try naming this file so users will know when to use it.
Also add verbs to your function names, to show what they are doing.